### PR TITLE
Feature/loop 735 section page

### PR DESCRIPTION
**Pull request by martinyde**
https://jira.itkdev.dk/browse/LOOP-735

- Nyeste  indhold view
- Section page content
- Paragraph til views

Contains missing config file for views that broke views ui functionality

### CODE REVIEW

File: config/sync/filter.format.os2loop_section.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+uuid: 3bbfff41-ca40-4b1d-ad01-9fdf51e518a9
+langcode: en
+status: true
+dependencies: {  }
+name: 'Section (rich text)'
+format: os2loop_section
```
Comment (rimi-itk) [inline config/sync/filter.format.os2loop_section.yml:6]: This should contain `os2loop_section_page`.

File: config/sync/editor.editor.os2loop_section.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+uuid: 86abbc66-e33a-4482-9ebf-129f66841866
+langcode: en
+status: true
+dependencies:
+  config:
+    - filter.format.os2loop_section
+  module:
+    - ckeditor
+format: os2loop_section
```
Comment (rimi-itk) [inline config/sync/editor.editor.os2loop_section.yml:9]: This should contain `os2loop_section_page`.

File: config/sync/editor.editor.os2loop_section.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+uuid: 86abbc66-e33a-4482-9ebf-129f66841866
```
Comment (rimi-itk) [inline config/sync/editor.editor.os2loop_section.yml:1]: The filename should contain `os2loop_section_page`.

File: config/sync/filter.format.os2loop_section.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+uuid: 3bbfff41-ca40-4b1d-ad01-9fdf51e518a9
```
Comment (rimi-itk) [inline config/sync/filter.format.os2loop_section.yml:1]: The filename should contain `os2loop_section_page`.

File: config/sync/user.role.authenticated.yml [hunk 5]
```diff
@@ -40,6 +40,7 @@ permissions:
   - 'use text format os2loop_question_answer_rich_text'
   - 'use text format os2loop_question_plain_text'
   - 'use text format os2loop_question_rich_text'
+  - 'use text format os2loop_section'
```
Comment (rimi-itk) [inline config/sync/user.role.authenticated.yml:43]: This should contain `os2loop_section_page`.

File: config/sync/views.settings.yml [hunk 6]
```diff
@@ -0,0 +1,46 @@
+display_extenders: {  }
```
Comment (rimi-itk) [inline config/sync/views.settings.yml:1]: Is this config related to the `views_ui` module? If yes, it must be removed.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Supposing the config namespace is `os2loop_section_page`, some config ids must be updated to include `os2loop_section_page` (rather than just `os2loop_section`).

File: config/sync/views.settings.yml (hunk 6 above)
Comment (martinyde) [reply config/sync/views.settings.yml:1]: As mentioned in description this file is needed for views ui to work.
It is related to the views module which is already activated.
web/core/modules/views/config/install/views.settings.yml 
So we miss it when enabling views_ui

File: config/sync/views.view.newest_content.yml [hunk 7]
```diff
@@ -0,0 +1,672 @@
...
+dependencies:
+  config:
+    - node.type.os2loop_documents_collection
+    - node.type.os2loop_documents_document
+    - node.type.os2loop_post
+    - node.type.os2loop_question
+  module:
+    - node
+    - user
+id: newest_content
```
[the remark this answers is not in the document]
Comment (martinyde) [reply config/sync/views.view.newest_content.yml]: Fixed

File: config/sync/filter.format.os2loop_section.yml (hunk 1 above)
Comment (martinyde) [reply config/sync/filter.format.os2loop_section.yml]: Fixed

File: config/sync/editor.editor.os2loop_section.yml (hunk 3 above)
Comment (martinyde) [reply config/sync/editor.editor.os2loop_section.yml]: Fixed

File: config/sync/editor.editor.os2loop_section.yml (hunk 2 above)
Comment (martinyde) [reply config/sync/editor.editor.os2loop_section.yml]: Fixed

File: config/sync/filter.format.os2loop_section.yml (hunk 4 above)
Comment (martinyde) [reply config/sync/filter.format.os2loop_section.yml]: Fixed

File: config/sync/user.role.authenticated.yml (hunk 5 above)
Comment (martinyde) [reply config/sync/user.role.authenticated.yml]: Fixed

**Review by rimi-itk — APPROVED** [review]
